docs/reference/platforms/macOS/index.rst

```diff
@@ -15,3 +15,29 @@ signed app bundle.
 
    app
    xcode
+
+Platform quirks
+===============
+
+Requirements cannot be provided as source tarballs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Briefcase *cannot* install packages published as source tarballs into a macOS app, even
+if the package is a pure Python package that would produce a ``py3-none-any`` wheel.
+This is an inherent limitation in the use of source tarballs as a distribution format.
```

Comment:
That's not really true; it's a limitation in Briefcase.

Reply:
I'll stand by this statement. It's not possible to tell purely from a tarball whether a package will produce a binary or non-binary wheel, or to establish anything about the build environments that are supported by the project. That's an inherent limitation in source tarballs as a distribution format for Python artefacts.